NewSpring/crete
# feat: Added new isInReadMyBible Check

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds a check to see if a person is in the Read My Bible group to any `Person` query.

### How do I test this PR?
Run this query and make check to see what comes back in `isInReadMyBible`. If you are a part of the group it should come back `true` and if you're not part of the group it should come back `false`.

```
{
  currentUser {
    id
    profile {
      id
      firstName
      lastName
      nickName
      isInReadMyBible
      campus {
        name
      }
		}
  }
}
```

## TODO


## REVIEW


---

> The purpose of a PR Review is to _improve the quality of the software._
